odbc/src/api/odbc_uri.rs

```diff
@@ -14,6 +14,8 @@ const SSL: &[&str] = &["ssl", "tls"];
 
 lazy_static! {
     static ref KEYWORDS: RegexSet = RegexSetBuilder::new([
+        "^APP_NAME$",
```

Comment:
Are these 2 different options or just 2 synonyms?
If they are synonyms then which one takes precedence? 
Also, I suppose that this PR is not totally ready/done because then the `appName` needs to be propagated to the MDB client, isn't it?

Reply:
They are synonyms.  Since it's going through the `remove` function, the first value that is in the connection string is used.
The implementation to add it to the MDB client was already done [here](https://github.com/mongodb/mongo-odbc-driver/blob/be876b0cb7317670b6616eeb750addbc61a9b10b/odbc/src/api/functions.rs#L848), the missing part was adding it to the keyword list.

Reply:
Perfect!